types/puppeteer/index.d.ts

```diff
@@ -0,0 +1,378 @@
+  evaluateOnNewDocument(
+    fn: EvaluateFn<string>,
+    ...args: object[]
+  ): Promise<void>;
+
+  // Argument `fn` can be an arbitrary function
+  exposeFunction(name: string, fn: any): Promise<void>;
+
+  focus(selector: string): Promise<void>;
+  frames(): Frame[];
+  goBack(options?: Partial<NavigationOptions>): Promise<Response>;
+  goForward(options?: Partial<NavigationOptions>): Promise<Response>;
+  goto(url: string, options?: Partial<NavigationOptions>): Promise<Response>;
+  hover(selector: string): Promise<void>;
+  keyboard: Keyboard;
+  mainFrame(): Frame;
+  mouse: Mouse;
+  pdf(options?: Partial<PDFOptions>): Promise<Buffer>;
+  plainText(): Promise<string>;
+  press(key: string, options?: { text: string; delay: number }): Promise<void>;
+  reload(options?: NavigationOptions): Promise<Response>;
+  screenshot(options?: ScreenshotOptions): Promise<Buffer>;
+  setContent(html: string): Promise<void>;
+  setCookie(...cookies: Cookie[]): Promise<void>;
+  setExtraHTTPHeaders(headers: Headers): Promise<void>;
+  setJavaScriptEnabled(enabled: boolean): Promise<void>;
+  setRequestInterceptionEnabled(value: boolean): Promise<void>;
+  setUserAgent(userAgent: string): Promise<void>;
+  setViewport(viewport: Viewport): Promise<void>;
+  tap(selector: string): Promise<void>;
+  touchscreen: Touchscreen;
+  tracing: Tracing;
+  type(text: string, options?: { delay: number }): Promise<void>;
+  viewport(): Viewport;
+}
+
+export interface Browser {
+  close(): Promise<void>;
+  newPage(): Promise<Page>;
+  version(): Promise<string>;
+  wsEndpoint(): string;
+}
+
+export interface LaunchOptions {
+  /** Whether to ignore HTTPS errors during navigation. Defaults to false. */
+  ignoreHTTPSErrors: boolean;
+  /** Whether to run Chromium in headless mode. Defaults to true. */
+  headless: boolean;
+  /**
+   * Path to a Chromium executable to run instead of bundled Chromium. If
+   * executablePath is a relative path, then it is resolved relative to current
+   * working directory.
+   */
+  executablePath: string;
+  /**
+   * Slows down Puppeteer operations by the specified amount of milliseconds.
+   * Useful so that you can see what is going on.
+   */
+  slowMo: number;
+  /**
+   * Additional arguments to pass to the Chromium instance. List of Chromium
+   * flags can be found here.
+   */
+  args: string[];
+  /** Close chrome process on Ctrl-C. Defaults to true. */
+  handleSIGINT: boolean;
+  /**
+   * Maximum time in milliseconds to wait for the Chrome instance to start.
+   * Defaults to 30000 (30 seconds). Pass 0 to disable timeout.
+   */
+  timeout: number;
+  /**
+   * Whether to pipe browser process stdout and stderr into process.stdout and
+   * process.stderr. Defaults to false.
+   */
+  dumpio: boolean;
+  /** Path to a User Data Directory. */
+  userDataDir: string;
+}
+
+export function launch(options?: LaunchOptions): Promise<Browser>;
+export function myOtherMethod(a: number): number;
```

Comment:
Did you mean to leave this in here?

Reply:
Oh shoot, totally missed that!

Reply:
@uniqueiniquity Fixed now. Thanks again for your review 👍